app/views/catalog/_collection_overview.html.erb

```diff
@@ -1,7 +1,15 @@
-<h2 class="sr-only">Collection description</h2>
-<% unless blacklight_config.show.metadata_partials.nil? %>
-  <% blacklight_config.show.metadata_partials.each do |metadata| %>
-    <% next unless fields_have_content?(@document, metadata) %>
-    <%= render partial: 'custom_metadata', locals: { document: @document, field_accessor: metadata } %>
-  <% end %>
-<% end %>
+<div class="row">
+  <div class="col-md-3 order-md-10">
```

Comment:
```suggestion
  <div class="col-md-3 order-md-2">
```
This should be ok as an `order-md-2` since this div is a sibling of the collection description. Can we change this class name so it's a little clearer semantically? I was scanning the HTML in-browser and went looking for 9 siblings 😅

Reply:
@camillevilla Aha, yes, good eye! This looks like a BS4ism that I was expecting to work like push/pull.